tests/ops/api/v1/endpoints/test_connection_config_endpoints.py

```diff
@@ -1365,14 +1365,17 @@ def test_put_connection_config_secrets_schema_validation(
         self, url, api_client: TestClient, generate_auth_header, connection_config
     ) -> None:
         auth_header = generate_auth_header(scopes=[CONNECTION_CREATE_OR_UPDATE])
-        payload = {"incorrect_postgres_uri_component": "test-1"}
+        payload = {
+            "host": "host.docker.internal",
+            "dbname": "postgres_example",
+            "incorrect_postgres_uri_component": "test-1",
+        }
         resp = api_client.put(
             url,
             headers=auth_header,
             json=payload,
         )
-        assert resp.status_code == 422
-        assert json.loads(resp.text)["detail"][0]["msg"] == "extra fields not permitted"
+        assert resp.status_code == 200
```

Comment:
This should return a 200 now that we ignore extra fields instead of forbid them.